tutorials/dlp-hybrid-inspect/index.md

```diff
@@ -111,27 +120,98 @@ Passwords are sent using Cloud Secret Manager to avoid exposing them through the
 | `sql`                | Database type: `"mysql"`, `"postgres"`, or `"cloudsql"`. There must be an appropriate JDBC driver configured in your `pom.xml` file. |
 | `threadPoolSize`     | Number of worker threads. This code uses 1 thread per table regardless of this setting. If you are scanning multiple tables, increasing this number means that more than one table can be processed in parallel. |
 | `sampleRowLimit`     | Maximum number of rows to scan per table.   |
-| `hybridJobName`      | Cloud DLP Hybrid job resource ID/name.      |
-| `databaseInstanceDescription` | An instance name for tracking. This name gets written to Hybrid and must follow label contraints. |
+| `hybridJobName` or `hybridJobTriggerName`      | Cloud DLP Hybrid job or trigger resource ID/name. You need to specify either a HybridDlpJob Id or a HybridJobTrigger Id. If both are specified then only one will be used.      |
+| `databaseInstanceDescription` | An instance name for tracking. This name gets written to Hybrid and must follow label constraints. |
 | `databaseInstanceServer`    | The hostname where your database is running (for example, `localhost` or `127.0.0.1` for local). |
 | `databaseName`              | The name of the database or dataset name that you want to scan. |
 | `tableName`                 | (Optional) The name of the table you want to scan. If blank, then all tables will be scanned. |
 | `databaseUser`              | The username for your database instance.  |
 | `secretManagerResourceName` | The Secret Manager resource ID/name that has your database password. Currently this is just the _name_ and the secret must be in the same project. |
 
-## Example command line for MySQL
+#### Example command line for MySQL
 
     java -cp target/dlp-hybrid-inspect-sql-0.5-jar-with-dependencies.jar com.example.dlp.HybridInspectSql \
-    -sql "mysql" \
+    -sql "postgres" \
     -threadPoolSize 1 \
     -sampleRowLimit 1000 \
     -hybridJobName "projects/[PROJECT_ID]/dlpJobs/[HYBRID_JOB_NAME]" \
     -databaseInstanceDescription "Hybrid Inspect Test" \
-    -databaseInstanceServer "127.0.0.1" \
+    -databaseInstanceServer "10.0.0.20" \
     -databaseName "[DATABASE_NAME]" \
     -databaseUser "[DATABASE_USER]" \
     -secretManagerResourceName "[SECRET_MANAGER]"
 
+### Option 2: Scan a list of data sources provided in a JSON file
+
+You can configure parameters including JSON file specifying a list of data sources to scan and what Cloud DLP hybrid job ID to use.
+Passwords are sent using Cloud Secret Manager to avoid exposing them through the command line.
```

Comment:
Should this be made the only option? You can simplify a lot if you don't need to support two means of specifying this information. Scripts with huge number of options get awkward anyway, when you have a group of related required options like this bundling them into a config file isn't a bad idea anyway.

Reply:
I'm mixed on this one. I can see some value here in both options but also see the complexity.  Maybe we leave this in for now in case anyone is using option 1 in the current version?

Reply:
I guess it depends on how you're treating this script. If it's essentially a demo, that we don't intend for customers to use as-is in a prod environment, we're definitely OK not being backwards compatible. It's also ideal for a demo to be as simple as possible. The github repo this is hosted in, as well as the directory structure, would probably lead someone finding this to assume this is the case. 

If this is intended as a turnkey prod solution for customers, and we already have anyone using it, then we should avoid breaking them... but it would also mean we're probably hosting the script in the wrong place.

Reply:
I think for now I'll leave both in and we'll see if it makes sense to cut later.